app/models/recipe.py

```diff
@@ -1,12 +1,26 @@
 import uuid
 from pydantic import BaseModel, root_validator
 from pydantic.types import UUID4
+# maybe "from ingredient import Ingredient" or is it "from app.models.ingredient import Ingredient"?
+# not sure how many folders back I need to go, if any, since the Recipe and Ingredient classes are in the same folder
 
 
 class Recipe(BaseModel):
     id: UUID4
     name: str
+    recipeYield: str
     description: str
+    ingredientList: str  # I assume this should be integrated with the ingredient model?
+
+    """
+        i'll leave the list at that for now but there are other possible items such as:

```

Comment:
Should file these as "issues" for future work, ideally todos aren't left in the code itself.

Reply:
Moved those comments to the issue tab.
https://github.com/DFNPLP/CrudExcercise/issues/2